api/envoy/config/wasm/v3alpha/wasm.proto

```diff
@@ -39,6 +37,8 @@ message VmConfig {
 message PluginConfig {
   // A unique name for a filters/services in a VM for use in identifiying the filter/service if
   // multiple filters/services are handled by the same vm_id and root_id and for logging/debugging.
+  // If left blank the plugin will be assigned an integer name corresponding to when it registers
+  // with the VM (i.e. the first plugin to register with the VM will be "0").
```

Comment:
Why? Is it only to identify them? UUID might be better, then, and it's already used for that purpose in Envoy.

Reply:
Removed it.  This is purely user-facing so if they don't add one then they done need it.